tests/unit/test_models/test_full_battery_models/test_lithium_ion/test_electrode_soh.py

```diff
@@ -2,11 +2,22 @@
 # Tests for the lithium-ion electrode-specific SOH model
 #
 
+import pytest
 import pybamm
-import unittest
 
 
-class TestElectrodeSOH(unittest.TestCase):
+@pytest.fixture()
+def options():
+    options = {
+        "open-circuit potential": "MSMR",
+        "particle": "MSMR",
+        "number of MSMR reactions": ("6", "4"),
+        "intercalation kinetics": "MSMR",
+    }
+    return options
+
```

Comment:
Could you move this fixture down to just above `TestElectrodeSOHMSMR`, so that we know it is relevant only for those tests?

Reply:
Either that, or (maybe better) you can add in a comment here the names of the classes that this applies to.